composer.json

```diff
@@ -25,7 +25,7 @@
     ],
     "require": {
         "php": "^7.1",
-        "guzzlehttp/guzzle": "^6.2",
+        "guzzlehttp/guzzle": "^6.2|^7.0",
```

Comment:
Based on Composer's documentation, we should use a double pipe for logical OR with regards to version ranges. I have created a pull request for this.